mod/system_log/lib/deprecated.php

```diff
@@ -0,0 +1,136 @@
+<?php

```

Comment:
do we need this deprecated? without the plugin active deprecation notices will never be triggered...

Reply:
Are you suggesting we just remove them and put into docs?

Reply:
yeah... i think so... don't think these functions will be called a lot directly